README.md

```diff
@@ -82,6 +84,47 @@ or as admin:
 - username: `tech@gisaia.com`
 - password: `admin`
 
+# ARLAS AIAS deployment
+
+The AIAS (ARLAS Item and Asset Services) deployment has:
+- [apisix](https://apisix.apache.org/)
+- [arlas-wui-iam](https://github.com/gisaia/ARLAS-wui-iam)
+- [arlas-iam-server](https://github.com/gisaia/ARLAS-IAM)
+- [postgres](https://www.postgresql.org/)
+- [arlas-wui](https://github.com/gisaia/ARLAS-wui)
+- [arlas-hub](https://github.com/gisaia/ARLAS-wui-hub)
+- [arlas-builder](https://github.com/gisaia/ARLAS-wui-builder)
+- [arlas-persistence-server](https://github.com/gisaia/ARLAS-persistence)
+- [arlas-permissions-server](https://github.com/gisaia/ARLAS-permissions)
+- [arlas-server](https://github.com/gisaia/ARLAS-server)
+- [agate](https://github.com/gisaia/aias)
+- [fam](https://github.com/gisaia/aias)
+- [fam-wui](https://github.com/gisaia/aias)
+- [aproc-service](https://github.com/gisaia/aias)
+- [aproc-proc](https://github.com/gisaia/aias)
+- [elasticsearch](https://github.com/elastic/elasticsearch)
+- [protomaps](https://protomaps.com/)
+- [minio](https://min.io)
+- [redis](https://redis.io)
+- [rabbitmq](https://www.rabbitmq.com)
+
+To start, run: 
+```shell
+./start.sh aias
+```
+
+You can access ARLAS just like the IAM deployement. You can also use the same script for initializing the stack with users and data.
+
+To setup an ARLAS EO catalog:
+1) place some geotiff files in ${APROC_INPUT_DIR} configured in [conf/aias.env](conf/aias.env). 
+2) go to the **Import** page (top right menu) to add the geotiff file in the catalog.
```

Comment:
What Import page ? It does not seem very clear

Reply:
Indeed, I added some details.